ui/src/components/ServiceMapModal/ServiceMapModal.tsx

```diff
@@ -180,33 +190,44 @@ export const ServiceMapModal: React.FC<ServiceMapModalProps> = ({ isOpen, onOpen
                         <img alt="spinner" src={spinnerImg} style={{ height: 50 }} />
                     </div>}
                     {!isLoading && <div style={{ height: "100%", width: "100%" }}>
+                    <div style={{ display: "flex", justifyContent: "space-between" }}> 
+                        <div>
                         <Button
+                            startIcon={<img src={reset} className="custom" alt="reset" style={{ marginRight:"8%"}}></img>}
+                            size="large"
                             variant="contained"
-                            className={commonClasses.button}
-                            style={{ marginRight: 25 }}
-                            onClick={() => onClose()}
-                        >
-                            Close
-                        </Button>
-                        <Button
-                            variant="contained"
-                            className={commonClasses.button}
-                            style={{ marginRight: 25 }}
+                            className={commonClasses.outlinedButton + " " + commonClasses.imagedButton}
+                            style={{ marginRight: 25, paddingTop: "3px", paddingBottom: "1px"}}
                             onClick={resetServiceMap}
-                        >
+                        >                          
                             Reset
                         </Button>
                         <Button
+                            startIcon={<img src={refresh} className="custom" alt="refresh" style={{ marginRight:"8%"}}></img>}
+                            size="medium"
                             variant="contained"
-                            className={commonClasses.button}
+                            className={commonClasses.outlinedButton + " " + commonClasses.imagedButton}
                             onClick={refreshServiceMap}
                         >
                             Refresh
                         </Button>
+                        </div>
+                        <img src={close} alt="close" onClick={() => onClose()} style={{cursor:"pointer"}}></img>
+                    </div>
                         <Graph
                             graph={graphData}
                             options={ServiceMapOptions}
                         />
+                        <div className='legend-scale'>
+                            <ul className='legend-labels'>
```

Comment:
Are the colors the same colors as we have on the side of the traffic viewer?
if they are, can we change the color to be const?

Reply:
I can do that, its just Gustavo wasnt sure if to change the colors to Javier design. its decided not to change at the moment.